tests/constbuffer_ut/constbuffer_ut.c

```diff
@@ -1550,7 +1550,7 @@ TEST_FUNCTION(CONSTBUFFER_CreateFromOffsetAndSizeWithCopy_unhappy_path) /*this f
 }
 
 /*small tests that check code integrity*/
-TEST_FUNCTION(CONSBUFFER_HANDLE_serialization_constants)
+TEST_FUNCTION(CONSBUFFER_HANDLE_serialization_constants) // no-srs

```

Comment:
> // no-srs [](http://example.com/codeflow?start=56&length=10)

This should have a spec tagged. If there is no spec defining these constants, add a spec for it